### DISCUSSION TITLE
Join forces with another project

### DISCUSSION THREAD

**Discussion by EliranTurgeman** [CarlosCuezva/dashboards-Grafana-Teslamate#6] [Ideas]
While searching for additional dashboards for TeslaMate I saw your project and another good one:

https://github.com/jheredianet/Teslamate-CustomGrafanaDashboards

I suggest to join both project into one 💪🏼
🙂

**Comment by CarlosCuezva** [comment]
Hello Eliran!

First of all, thanks for your comment. Juan Carlos and I already talked about it some time ago and maybe in a future we join our projects, but not for now. We've both been collaborating on each other's repository.

Best regards and I hope you liked the dashboards. I'll really apreciate any improvement or suggestion.

**Reply by natrlhy** [reply]
Looking forward to trying these out!

**Reply by CarlosCuezva** [reply]
Sorry [natrlhy](https://github.com/natrlhy), but finally it isn't possible to join both projects.
